.editorconfig

```diff
@@ -211,4 +211,127 @@ dotnet_diagnostic.CA5401.severity = warning
 # Use CreateEncryptor with the default IV 
 dotnet_diagnostic.CA5402.severity = warning
 # Do not hard-code certificate
-dotnet_diagnostic.CA5403.severity = warning
\ No newline at end of file
+dotnet_diagnostic.CA5403.severity = warning

```

Comment:
The changes are added by Visual Studio automatically when I add the `.editorconfig` file as a solution item.